addon/webextension/background.js

```diff
@@ -8,10 +8,50 @@
 
 
 // constants used by particular effects
-const WORDS = "dark army disintegration data privacy internet delete".split(/\s+/);
-const XHEADERSITES = ['<all_urls>'];
-const XHEADERNAME = 'dontdeleteme';
-const XHEADERVALUE = '1057'
+const WORDS =
+`
+puppet
+one
+zero
+congo
+rewind
+fuck
+71
+fsociety
+encrypt
+decrypt
+control
+illusion
+dark
+army
+evil
+robot
+disintegration
+hack
+society
+white
+rose
+revolution
+subroutine
+backdoor
+undo
+society
+corporation
+economy
+mask
+system
+truth
+debt
+cryptocurrency
+kernel panic
```

Comment:
Can somebody explain this to me? Our regex splits on `\s+`, so wouldn't this parse as two separate words anyways? Or should we update the regexp to split on `\n`?

Reply:
It will go on both words as single words.  I decided not to be more clever than that.